requirements.txt

```diff
@@ -1,7 +1,7 @@
 # temporary until pystun creates a new release
 git+https://github.com/konradkonrad/pystun@develop#egg=pystun
 # temporary until new version of pyethereum is released, that supports interfaces
-git+https://github.com/LefterisJP/pyethereum@take_solidity_interface_into_account#egg=ethereum
+git+https://github.com/palango/pyethereum@take_solidity_interface_into_account#egg=ethereum
```

Comment:
What is missing to use the mainstream?

Reply:
See comment below, I'll look up the exact problem again.